### ISSUE TITLE
A11y_VSCode_DebugConsole_ScreenReader: Voice over does not announce the pop up title as 'Debug console' when it is opened using ctrl+shift+y.

### ISSUE THREAD

**Issue by bhagwatshelke1047** [microsoft/vscode#77056]
**Environment Details:**
VSCode Version : 1.36.1 (user setup)
OS Version : MacOS Mojave 10.14.5

**Additional Details:**
MAS Violated : MAS 1.3.1
Screen Reader Versions : Voice over

**Repro Steps:**
1) Open Visual studio code editor  
2) Press ctrl+shift+y
3) Validate that debugging messages show-up in Debug console
4) Start voice over

**Actual:**
Voice over does not announce the pop up title as 'Debug console' when it is opened using ctrl+shift+y.

This issue is observed in Mac + voice over

**Expected:**
Screen reader should announce the pop up title as 'debug console' when it is opened using ctrl+shift+y.

**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

**User Impact:**
Screen reader users will not get that on which page it is being redirected.

**MAS Reference:
MAS 1.3.1 -** https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

**Attachment for Reference:**
[A11y_VSCode_DebugConsole_ScreenReader.pptx](https://github.com/microsoft/vscode/files/3376156/A11y_VSCode_DebugConsole_ScreenReader.pptx)

**Comment by isidorn** [comment]
Dup of https://github.com/microsoft/vscode/issues/77063
